e2e/cephfs.go

```diff
@@ -298,35 +298,32 @@ var _ = Describe(cephfsType, func() {
 				})
 			}
 			By("verify generic ephemeral volume support", func() {
-				// generic ephemeral volume support is beta since v1.21.
-				if k8sVersionGreaterEquals(f.ClientSet, 1, 21) {
```

Comment:
same changes are required in rbd.go also.

Reply:
sure.. will club it in the same PR